src/index.js

```diff
@@ -0,0 +1,96 @@
+import './style.css';
+import addToDo from './modules/addFunction.js';
+import deleteTask from './modules/deleteFunction.js';
+import editTask from './modules/editFunction.js';
+import clearAllCheck from './modules/clearButton.js';
+import checkStatus from './modules/checkStatus.js';
```

Comment:
- [ ] While the approach of fragmenting functions into modules works, it is of most use when they're implemented as classes with their own methods to make the code smaller and more friendly to other developers. Kindly consider making your own objects for these functions.